### PR TITLE
Removes .rubocop.yml as no longer required

### LINKED ISSUE #406: Remove .rubocop.yml with Dangerfile
### :speaking_head: Foreword

Thank for taking the time to fill this bug report fully. Without it we may not be able to fix the bug, and the issue may be closed without resolution.

## :ghost: Brief Description

If the only thing in the `.rubocop/yml` is Dangerfile
Run the latest cookstyle
Remove `.rubpopo.yml`

Issue sprouted from sous-chefs/meta/issues/108. If not applicable then issue should be closed.

### PR DESCRIPTION
**Pull request by PanosCodes**
### Description

Removed .rubocop.yml as no longer required

### Issues Resolved
#406

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 This file is used to list changes made in each version of the haproxy cookbook.
 
+## [v8.1.2] (2019-10-10)
+
+ - Removed `.rubocop.yml` as no longer required
```
Comment (Xorima) [inline CHANGELOG.md]: Hey,

thanks for the Changelog entry, really appreciated, could you indent it at the start, that will fix the build errors in Markdown Lint
```suggestion
- Removed `.rubocop.yml` as no longer required
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 This file is used to list changes made in each version of the haproxy cookbook.
 
+## [v8.1.2] (2019-10-10)
```
Comment (JohnRoesler) [inline CHANGELOG.md]: Please make this heading unreleased. I won't be making a release for this as it's not changing the cookbook code
```suggestion
## Unreleased
```

File: metadata.rb [hunk 3]
```diff
@@ -3,7 +3,7 @@
 maintainer_email  'help@sous-chefs.org'
 license           'Apache-2.0'
 description       'Installs and configures haproxy'
-version           '8.1.1'
+version           '8.1.2'
```
Comment (JohnRoesler) [inline metadata.rb]: Please revert version to 8.1.1
```suggestion
version           '8.1.1'
```

**Comment by JohnRoesler** [comment]
Specifically lines [28](https://github.com/sous-chefs/haproxy/blob/master/Dangerfile#L28) and [34](https://github.com/sous-chefs/haproxy/blob/master/Dangerfile#L34)

File: CHANGELOG.md (hunk 2 above)
Comment (PanosCodes) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (PanosCodes) [reply CHANGELOG.md]: Done

File: metadata.rb (hunk 3 above)
Comment (PanosCodes) [reply metadata.rb]: Done

**Pull request merged**